libsql/src/replication/mod.rs

```diff
@@ -229,7 +255,20 @@ impl EmbeddedReplicator {
             }
         }
 
-        Ok(replicator.client_mut().committed_frame_no())
+        let last_frames_synced = self.last_frames_synced.fetch_add(
+            replicator.frames_synced(),
+            std::sync::atomic::Ordering::Relaxed,
```

Comment:
I usually recommend using sequential ordering unless you know this is a bottleneck in any way.

Reply:
While I don't disagree, I think the fact that this isn't being ordered with anything else + being a fetch_add means that these orderings don't mean much in the end and thus relaxed is fine.